src/pages/login-page/LoginPage.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useIntl } from "react-intl";
+import { Link, useNavigate } from "react-router-dom";
+
+import logo from "assets/logo.png";
+import LoginForm from "components/users/LoginForm/LoginForm";
+import { homePageLink } from "constants/linkConstants";
+
+import "./login-page.scss";
+
+const LoginPage = () => {
+  const intl = useIntl();
+
+  const navigate = useNavigate();
+
+  const { loading, loggedInSuccess } = useSelector((state) => state.user);
+
+  const forgotPasswordClass = loading ? "disable-link" : "";
```

Comment:
FYI: You can use [classnames](npmjs.com/package/classnames) to conditionally use of classNames